src/services/api/notebook/index.ts

```diff
@@ -0,0 +1,31 @@
+import EnvironmentVariables from 'helpers/EnvVariables';
+import { sendRequest } from 'services/api';
+import { TNotebookApiResponse } from './model';
+
+export const NOTEBOOK_API_URL = `${EnvironmentVariables.configFor('VARIANT_CLUSTER_API')}`;
+
+const start = () => {
+  return sendRequest<TNotebookApiResponse>({
+    method: 'POST',
```

Comment:
maybe use an enum for http methods.

Reply:
what would be the gain ?

Reply:
In this case, it is no big deal. But I think it is better to avoid raw strings in certain circumstances.